client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -37,12 +37,22 @@ class DesignTypeWithStoreStep extends Component {
 	getChoices() {
 		const { translate } = this.props;
 
-		return [
-			{ type: 'blog', label: translate( 'A list of my latest posts' ), image: <BlogImage /> },
-			{ type: 'page', label: translate( 'A welcome page for my site' ), image: <PageImage /> },
-			{ type: 'grid', label: translate( 'A grid of my latest posts' ), image: <GridImage /> },
-			{ type: 'store', label: translate( 'An online store' ), image: <StoreImage /> },
-		];
+		switch ( abtest( 'signupCopy' ) ) {
```

Comment:
Up to you, but `switch` seems a bit of an overkill? A simple `if`/`else` should be enough and show your intent better:
```
if ( abtest( 'signupCopy' ) === 'newSignupCopy' ) {
 // new copy
} else {
 // old copy
}
```